src/main/java/com/team701/buddymatcher/socket/ChatObject.java

```diff
@@ -0,0 +1,34 @@
+package com.team701.buddymatcher.socket;
+
+public class ChatObject {
+
+    private String userName;
+    private String message;
+
+    /**
+     * Just for testing of socket i/o, to be removed
+     */
```

Comment:
I'm a bit worried we'll forget about this and never remove it... perhaps create an issue to make sure we remember?

Reply:
Will do!